<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QSlider" name="throttle_slider">
      <property name="minimum">
       <number>700</number>
      </property>
      <property name="maximum">
       <number>1500</number>
      </property>
      <property name="singleStep">
       <number>4</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="throttle">
      <property name="text">
       <string>Throttle</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="rotation">
      <property name="text">
       <string>Rotation</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="rotation_slider">
      <property name="minimum">
       <number>600</number>
      </property>
      <property name="maximum">
       <number>1600</number>
      </property>
      <property name="singleStep">
       <number>4</number>
      </property>
      <property name="value">
       <number>1100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="elev">
      <property name="text">
       <string>Elev</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="elev_slider">
      <property name="minimum">
       <number>600</number>
      </property>
      <property name="maximum">
       <number>1600</number>
      </property>
      <property name="singleStep">
       <number>4</number>
      </property>
      <property name="value">
       <number>1100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="aile">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Aile</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSlider" name="aile_slider">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimum">
       <number>600</number>
      </property>
      <property name="maximum">
       <number>1600</number>
      </property>
      <property name="singleStep">
       <number>4</number>
      </property>
      <property name="value">
       <number>1100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="turnedOn">
      <property name="text">
       <string>On/Off</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>turnedOn</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>turnedOn(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>throttle_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateThrottle(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotation_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>elev_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateElev(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aile_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateAile(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>turnedOn(bool)</slot>
  <slot>updateThrottle(int)</slot>
  <slot>updateRotation(int)</slot>
  <slot>updateElev(int)</slot>
  <slot>updateAile(int)</slot>
 </slots>
</ui>
